pkg/tfvars/openstack/openstack.go

```diff
@@ -193,3 +205,75 @@ func getServiceCatalog(cloud string) (*tokens.ServiceCatalog, error) {
 
 	return serviceCatalog, nil
 }
+
+// getImagesCacheDir returns a local path of image cache, where the installer should put the data:
+// <user_cache_dir>/openshift-installer/image_cache
+// If the directory doesn't exist, it will be automatically created.
+func getImagesCacheDir() (string, error) {
+	userCacheDir, err := os.UserCacheDir()
+	if err != nil {
+		return "", err
+	}
+
+	imagesCacheDir := filepath.Join(userCacheDir, "openshift-installer", "image_cache")
+
+	if _, err := os.Stat(imagesCacheDir); os.IsNotExist(err) {
```

Comment:
if we failed to stat for any reason other than `IsNotExist`, this would incorrectly succeed.